src/client/invite/Invite.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { FormattedMessage } from 'react-intl';
+import { Form, Input } from 'antd';
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+import Affix from '../components/Utils/Affix';
+import LeftSidebar from '../app/Sidebar/LeftSidebar';
+import requiresLogin from '../auth/requiresLogin';
+import { getAuthenticatedUserName } from '../reducers';
+import './Invite.less';
+
+@requiresLogin
+@connect(
+  state => ({
+    authenticatedUserName: getAuthenticatedUserName(state),
+  }),
+)
+export default class Invite extends React.Component {
+  static propTypes = {
+    authenticatedUserName: PropTypes.string,
+  };
+
+  static defaultProps = {
+    authenticatedUserName: '',
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
We usually initialize state like this:
https://github.com/busyorg/busy/blob/master/src/client/components/AvatarLightbox.js#L20-L22